tools/data_sources/README.md

```diff
@@ -0,0 +1,31 @@
+# Data Commons Data Sources Generation Tool
+
+## Purpose
+
+This script is designed to scrape the existing data sources pages (external to)
+the primary site, and convert those data sources to json in a format that 
+is usable by the data sources page in the primary Data Commons site.
+
+## Notes
+
+This script may be temporary. The external data sources pages are populated
```

Comment:
could you please update this to a TODO for us to migrate into updating this page directly - thanks!